doc/2-interface/settings.md

```diff
@@ -356,6 +356,10 @@ below all the binds, select a key from the dropdown list to add it. it will appe
   - **/path/to/file.fur - Furnace**
 - **Display system name on title bar**
 - **Display chip names instead of "multi-system" in title bar**
+- **Export options layout:**
+  - **Sub-menus in File menu**: splits the "export..." menu item into one item for each export type. selecting one opens a popup with settings.
```

Comment:
Whaaaaat?

Reply:
How would you describe it?

Reply:
How about this?

- **Export options layout:**
  - **Sub-menus in File menu**: each export type has its own "export" menu entry. selecting one shows a popup for settings.
  - **Modal window with tabs**: the "export..." menu opens a window with a tab for each export type. default.
  - **Modal windows with options in File menu**: each export type has its own "export" menu entry. selecting one opens a window for settings.

Reply:
Looks better - will cherry-pick.